.gitignore

```diff
@@ -0,0 +1,5 @@
+*/Pods/
```

Comment:
remark(no-blocking) : you can use some tools to generate the gitignore : https://www.toptal.com/developers/gitignore

Reply:
I used this website as reference actually, but as the projet organised differently(there is ObjC and Swift as parent folder + one share workspace for Admob and direct integration) needed to customise a little bit :).

I will do the update for the ObjC version & Readme before merge this PR,
thanks in advance :)